### PR TITLE
♻️(site) Integration and content touchups

### LINKED ISSUE #295: Article Detail: News
Design: https://www.figma.com/file/xtBKZcRBo7H2Jrjs7DjNqE/Final-Designs?node-id=2%3A23765

TO DO:
Per Sam comment on PR #349 
- move author/date information in a `footer` tag
- keep table of contents an `aside` tag

### LINKED ISSUE #296: Article Detail: Stories
Design: https://www.figma.com/file/xtBKZcRBo7H2Jrjs7DjNqE/Final-Designs?node-id=2%3A21311

### LINKED ISSUE #347: Module: Hero (Stories)
Stories Landing Page :: Hero

Design: https://www.figma.com/file/xtBKZcRBo7H2Jrjs7DjNqE/Final-Designs?node-id=2%3A26009

### PR DESCRIPTION
**Pull request by Snugug**
- [x] Allows stats to have a decimal
- [x] Adds Codecademy
- [x] Update featuring logic
    * Logic to determine featured stories on home and story landing page
    * Media grab for story cards
    * Featured Story component to use media instead of featured images
- [x] Updates blog post large screen layout
    * Update article layout
    * Update article footer
- [x] Adds EE option to not switch to theme immediately
- [x] Normalizes story and post tags
    * Web/PWA
    * Translations
    * Eyebrows for Stories
- [x] Adds banner overrides
- [x] Fixes Simplifi image on landing page
- [x] Adds banners to Crostini post
- [x] Adds hero banner background image direction shift
- [x] Small tweaks to Phosphor
- [x] Makes converted GIFs more performant by removing the full download as a poster
- [x] Removed unneeded footnote section on homepage
- [x] Update navigation breakpoints
- [x] Update featured images for story cards that are no longer featured
- [x] Fix slight spacing issue between body and header

* Resolves #295 
* Resolves #347 
* Resolves #296

### CODE REVIEW

File: lib/transforms/google-storage-images.js [hunk 1]
```diff
@@ -61,7 +72,7 @@ module.exports = async function postHTMLGoogleStorageImage(tree) {
           muted: true,
           playsinline: true,
           crossorigin: 'annoymous',
-          poster: `${url}?auto=format,compress`,
+          // poster: `${url}?auto=format,compress`,
```
Comment (Vanss472) [inline lib/transforms/google-storage-images.js:75]: remove commented code
```suggestion
```

File: site/sass/components/_hero-article.scss [hunk 2]
```diff
@@ -5,11 +5,19 @@
   $content-max-width: 100rem;
 
   overflow: hidden;
+  // padding-bottom: clamp(1rem, 6vw, 2rem)
```
Comment (Vanss472) [inline site/sass/components/_hero-article.scss:8]: Do you need this? same on Line 55

**Review by Vanss472 — COMMENTED** [review]
Hey @Snugug this is looking good so far. I left a couple of comments, minor tweaks. Good job, thanks!

File: site/sass/components/_date-time.scss [hunk 3]
```diff
@@ -9,6 +9,10 @@
     margin-bottom: 0.5rem;
   }
 
+  &--inline {
+    margin-bottom: 0;
+  }
+
   &--inline & {
     &__label {
       display: inline;
```
[the remark this answers is not in the document]
Comment (Snugug) [reply site/sass/components/_date-time.scss:22]: Done!

**Review by Vanss472 — APPROVED** [review]
LGTM!

**Pull request merged**